src/include/transaction/deferred_action_manager.h

```diff
@@ -25,7 +27,6 @@ class DeferredActionManager {
       : timestamp_manager_(timestamp_manager) {}
```

Comment:
I would add a comment in the function description that says that if timestamp_manager is null then that means gc is off.